.gitignore

```diff
@@ -49,3 +52,6 @@ nohup.out
 
 # tests
 test/output/*
+
+# package-lock.json
+package-lock.json
```

Comment:
package-lock.json belongs in the repo. The point is that unlike package.json, it sets the specific versions for both our dependencies and their dependencies, or the whole dependency chain. That makes sure that whoever executes "yarn" (~npm install) while having the same package-lock.json file, gets the exact same packages (that are known to work).

=> Please revert this part.

Reply:
Actually there is no package-lock.json yet in the repo, but we could add it at some point. It was yarn.lock that I added yesterday... (yarn.lock for yarn, package-lock.json for npm)